package.json

```diff
@@ -29,8 +28,10 @@
     "rollup-plugin-terser": "^4.0.2",
     "semver": "^5.6.0"
   },
+  "peerDependencies": {
+    "chart.js": "^2.6.0"
+  },
   "dependencies": {
-    "chart.js": "^2.6.0",
     "hammerjs": "^2.0.8"
```

Comment:
The goal was to have it specified as a dependency so that npm would pull it in. We had to make it external so that it wouldn't be bundled and pulled in via npm. I think npm should manage the dependency so that if another dependency includes hammer it can be deduped. I don't think the user should have to specify hammer directly in their project though. So I do think `dependencies` is probably the right place to specify it